proxy-robots-module/src/main/java/orwell/proxy/ProxyRobots.java

```diff
@@ -201,9 +209,6 @@ public void receivedNewZmq(final ZmqMessageBOM zmqMessageBOM) {
         }
     }
 
-    protected int getNbOutgoingMessageFiltered() {
-        return outgoingMessageFiltered;
-    }
```

Comment:
I wanted to comment line 215 but it is out of scope.
The comment is not correct and the test could be extracted in a separate method.
This method could be split in smaller methods.